app/src/main/java/com/architect/coders/mu8/remote/model/getallComic/CharacterDTO.kt

```diff
@@ -0,0 +1,14 @@
+package com.architect.coders.mu8.remote.model.getallComic
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class CharacterDTO (
```

Comment:
Let's rename all `DTO` classes as `Response` for better understanding, in this case it will be `CharacterResponse`.